api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PersonAttributeTranslatorImpl.java

```diff
@@ -200,16 +200,14 @@ protected Reference buildLocationReference(String locationId) {
 	}
 	
 	protected CodeableConcept buildCodeableConcept(String conceptId) {
-		if (conceptId == null) {
-			return null;
```

Comment:
I'd prefer if we still had the null check here, I think.

Reply:
Thanks for the feedback @ibacher . I added the validation and kept the layout similar to what we have above for the location type.